Nodejs/HttpTriggerIL/index.js

```diff
@@ -0,0 +1,13 @@
+function in_loop(n) {
+    let x = 0, y = 0;
+
+    for (var i = 0; i < n; i++) {
+        x = 1 + 1;
+        y = y + x;
+    }
+}
+
+module.exports = async function (context, req) {
+    context.log('JavaScript HTTP trigger function processed a request.');
```

Comment:
This could be a source of noise since it involves I/O

Reply:
Removed